source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst

```diff
@@ -0,0 +1,726 @@
+.. _bare-metal-install-beta:
+
+Install |CL-ATTR| on bare metal
+###############################
+
+These instructions guide you through installation of |CL-ATTR| on bare metal 
+using the new installer. 
+
+.. contents:: :local:
+   :depth: 1
+
+.. _minimum-installation-requirements: 
+
+Minimum installation requirements
+*********************************
+
+To fulfill minimum installation requirements, complete 
+``Required options``. All other menu items are optional.
+
+.. note::
+
+   * In the Main menu, the :kbd:`Install` button is highlighted 
+     **only after** you complete :ref:`required-options-installation`.
+
+   * You must choose whether or not to participate in :ref:`telemetrics`
+     before you can :ref:`complete-cl-installation`.
+
+   * You may wish to :ref:`test-network-settings` before you 
+     :ref:`configure-network-interfaces`. Assure that a ``Success`` message is received.
+
+Main menu
+*********
+
+In the |CL| Installer main menu, two tabs appear: 
+:guilabel:`[R] Required options` and :guilabel:`[A] Advanced options`.
+Navigate between tabs using the arrow keys or these shortcut keys:
+
+* :kbd:`Shift+A` for :guilabel:`[A] Advanced options`
+* :kbd:`Shift+R` for :guilabel:`[R] Required options`
+
+To meet the minimum requirements, enter your choices in the 
+:guilabel:`Required options`. After confirmation, your selections appear 
+below menu options in the Main Menu, beside the :guilabel:`>>` chevron.
+
+Navigation
+**********
+
+* Select :kbd:`Tab` or :kbd:`Up/Down` arrows to highlight your choice.
+
+* Select :kbd:`Enter` or :kbd:`Spacebar` to confirm your choice.
+
+* Select :kbd:`Cancel` or :kbd:`Esc` to cancel your choice.
+
+  The |CL| Installer main menu appears as shown in Figure 1.
```

Comment:
@krisdale . I'm considering scaling up all images in this doc--probably to 75%. When one builds this, the images appear too small. Please let me know your thoughts on this.